src/services/recipient.js

```diff
@@ -511,6 +512,14 @@ export async function getGoalsByActivityRecipient(
     ...filters
   },
 ) {
+  // Get the GoalTemplateFieldPrompts where title is 'FEI root cause'.
+  const feiRootCauseFieldPrompt = await GoalTemplateFieldPrompt.findOne({
```

Comment:
can we cache this somehow? seems bad to do this query this every single time we make this request, considering the result will always be the same

Reply:
What are you thinking for cache?

Reply:
redis, or in memory